app/src/main/java/net/skyscanner/backpack/demo/compose/CalendarStory.kt

```diff
@@ -0,0 +1,68 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import kotlinx.coroutines.flow.filter
+import net.skyscanner.backpack.calendar2.CalendarEffect
+import net.skyscanner.backpack.calendar2.CalendarSelection
+import net.skyscanner.backpack.compose.calendar.BpkCalendar
+import net.skyscanner.backpack.compose.calendar.rememberCalendarController
+import net.skyscanner.backpack.demo.data.CalendarStorySelection
+import net.skyscanner.backpack.demo.data.CalendarStoryType
+import net.skyscanner.backpack.toast.BpkToast
+
+@Composable
+fun CalendarStory(
+  type: CalendarStoryType,
+  modifier: Modifier = Modifier,
+) {
+  val context = LocalContext.current
+  val automationMode = LocalAutomationMode.current
+  val controller = rememberCalendarController(initialParams = CalendarStoryType.createInitialParams(type))
+
+  LaunchedEffect(type, controller, automationMode) {
+    when (type) {
+      CalendarStoryType.SelectionWholeMonth -> controller.setSelection(CalendarStorySelection.WholeMonthRange)
+      CalendarStoryType.PreselectedRange -> controller.setSelection(CalendarStorySelection.PreselectedRange)
+      else -> Unit
+    }
+
+    controller.state
+      .filter { it.selection !is CalendarSelection.None }
+      .collect {
+        if (!automationMode) {
+          BpkToast.makeText(context, it.selection.toString(), BpkToast.LENGTH_SHORT).show()
```

Comment:
Again, that was done for comparison of the screens. We can replace it with Snackbar, the only concern is that it may overcomplicate the same code. WDYT?

Reply:
snackbar is also essentially deprecated - floating notification is the part that should be used now.
Maybe we need to have a generic floating notification composable that we can use to show in stories

Reply:
Sounds great, I'll do this!

Reply:
Done